bccsp/handlers/nymsigner.go

```diff
@@ -35,6 +32,32 @@ func (s *NymSigner) Sign(k bccsp.Key, digest []byte, opts bccsp.SignerOpts) ([]b
 		return nil, errors.New("invalid issuer public key, expected *issuerPublicKey")
 	}
 
+	// handle the smartcard case
+	if signerOpts.IsSmartcard {
+		if s.SmartcardNymSignatureScheme == nil {
+			return nil, fmt.Errorf("smartcard mode is unsupported")
```

Comment:
let's always use `"github.com/pkg/errors"`, please.

Reply:
I mentioned this because in some other places `"github.com/pkg/errors"` is used.
Then, a possible solution would be to create a local `errors` package with aliases to the functions you want to use. Then, the code can be refactored to use those aliases.
Up to you :)

Reply:
I think the "right" choice would be to just follow the new go error proposal and start using just `fmt`. There are other active forks of `"github.com/pkg/errors"`, that's not even the issue. The issue is rather that if the proper way of handling errors is another one, we shouldn't use them. Let me investigate a bit more, but I think for now we can leave it as is?